sharpd/sharp_zebra.c

```diff
@@ -351,6 +345,49 @@ static int sharp_nexthop_update(ZAPI_CALLBACK_ARGS)
 			break;
 		}
 	}
+
+	return i;
+}
+static int sharp_nexthop_update(ZAPI_CALLBACK_ARGS)
+{
+	struct sharp_nh_tracker *nht;
+	struct zapi_route nhr;
+	char buf[PREFIX_STRLEN];
+
+	if (!zapi_nexthop_update_decode(zclient->ibuf, &nhr)) {
+		zlog_warn("%s: Decode of update failed", __PRETTY_FUNCTION__);
+
+		return 0;
+	}
+
+	zlog_debug("Received update for %s",
+		   prefix2str(&nhr.prefix, buf, sizeof(buf)));
```

Comment:
nit: here and in other places you could use `%pFX` to print prefixes directly (without `prefix2str()` and the local buffer).

Reply:
shit I forgot about those.

Reply:
Fixed